lib/pdk/cli/validate.rb

```diff
@@ -15,6 +15,7 @@ module PDK::CLI
 
     flag nil, :list, _('list all available validators')
     flag :a, 'auto-correct', _('automatically correct problems (where possible)')
+    flag nil, :parallel, _('run validations in parallel')
```

Comment:
For the upcoming community release, we'll need to comment this line and hardcode line 74 with `false`.